openapi/auth-server.yaml

```diff
@@ -532,6 +588,26 @@ components:
             - debitAmount
         - required:
             - receiveAmount
+    gnap-error:
```

Comment:
what do you think of having separate objects per error, and then using specific ones per-response?
e.g. we wouldn't have  `too_fast` in a normal grant request, just for grant continuation.

Maybe that is too specific, but curious to hear your opinion

Reply:
I considered that, but it felt at times overly granular to have a schema for an error that was only referenced once - `too_fast` is a pretty good example of that, as it's only thrown by the grant continuation endpoint.

In hindsight though, I think it would be more useful to use the specific ones per-response. Biggest reason to me being that these routes may throw the same code for different reasons, primarily 401 could be thrown by the signature middleware or the actual business logic...